CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [v4.1.0](http://github.com/ndustrialio/contxt-sdk-js/tree/v4.1.0) (2021-02-02)
+
+**Changed**
+
+- Adds `AssetMetrics#getValuesByMetricIdsAssetIds` to fetch multiple asset metrics for multiple asset ids within a specified date range.
```

Comment:
Can you change this from `fetch multiple asset metrics` to `fetch multiple asset metrics values`. And maybe capitalize the English version of IDs (that's just a nit)